### ISSUE TITLE
Missing prerequisites

### ISSUE THREAD

**Issue by chrismanuel1993** [MicrosoftDocs/powerbi-docs#3106]
[Enter feedback here]
Should also be added that to save a copy of a report (without an app method), in addition to build access, they also need to be in a non-Read-Only role  

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 83c63999-09de-cccf-ae53-ef7fcfbc1186
* Version Independent ID: 39407557-14d4-f78b-bfcb-ea95cf501c47
* Content: [Copy reports from other apps or workspaces - Power BI](https://docs.microsoft.com/en-us/power-bi/connect-data/service-datasets-copy-reports)
* Content Source: [powerbi-docs/connect-data/service-datasets-copy-reports.md](https://github.com/MicrosoftDocs/powerbi-docs/blob/master/powerbi-docs/connect-data/service-datasets-copy-reports.md)
* Service: **powerbi**
* Sub-service: **pbi-data-sources**
* GitHub Login: @paulinbar
* Microsoft Alias: **painbar**

**Comment by KesemSharabi** [comment]
Thanks for your comment, @[github person who posted].

I've assigned this issue to the doc author for review.

**Comment by paulinbar** [comment]
Hi @chrismanuel1993, thanks for your comment. Yes, you're right, you need at least a Contributor role in the workspace where the report resides. I'm working to clarify in the document. Thanks!

**Comment by paulinbar** [comment]
@chrismanuel1993, as a matter of fact, if you are a Contributor in the Workspace where the report is located, you don't even need Build permission in addition. The Contributor role is enough.

**Comment by chrismanuel1993** [comment]
@paulinbar true. I think Contributor has build permission as default as they can edit

**Comment by paulinbar** [comment]
@chrismanuel1993, yes. In any event, I have engaged the product team to help me sort things out... when you scratch under the surface it starts to get complicated... so I'll get back to you soon.

**Comment by paulinbar** [comment]
Article has been updated.
